konduit-serving-pipeline/src/main/java/ai/konduit/serving/pipeline/impl/step/ml/ssd/SSDToBoundingBoxRunner.java

```diff
@@ -52,37 +57,43 @@ public Data exec(Context ctx, Data data) {
 
         String key = "detection_boxes";     //TODO
         String prob = "detection_scores";
+        String labels = "detection_classes";
 
         NDArray bND = data.getNDArray(key);
         NDArray pND = data.getNDArray(prob);
+        NDArray lND = data.getNDArray(labels);
 
         float[][][] bArr = bND.getAs(float[][][].class);        //Batch, num, xy
         float[][] pArr = pND.getAs(float[][].class);            //Batch, num
-
+        float[][] lArr = lND.getAs(float[][].class);
         List<BoundingBox> l = new ArrayList<>();
-        for(int i=0; i<bArr[0].length; i++ ){
+        for (int i = 0; i < bArr[0].length; i++) {
             //SSD order usually: [y1, x1, y2, x2]
             double y1 = bArr[0][i][0];
             double x1 = bArr[0][i][1];
             double y2 = bArr[0][i][2];
             double x2 = bArr[0][i][3];
             double p = pArr[0][i];
-            if(p >= threshold) {
-                l.add(BoundingBox.createXY(x1, x2, y1, y2, null, p));
+            float label = lArr[0][0];
+            if (step.COCO_CLASSES_LABELS.length > 0 && (int) label > step.COCO_CLASSES_LABELS.length){
+                throw new ArrayIndexOutOfBoundsException(String.format("Predicted label index was %s but only %s labels were provided", (int) label, step.COCO_CLASSES_LABELS.length));
+            }
+            if (p >= threshold) {
+                l.add(BoundingBox.createXY(x1, x2, y1, y2, !ArrayUtils.isEmpty(step.COCO_CLASSES_LABELS) ? step.COCO_CLASSES_LABELS[(int) label - 1] : "no label provided", p));
```

Comment:
All of these references to the COCO dataset have to be removed entirely from here.
The SSD architecture can be used with any datasets; COCO is just one possible dataset. It's like saying "all image 

So again, I'll repeat my earlier comment:
> Two things to change here
(a) the labels should come from the configuration (i.e., something the user sets in SSDToBoundingBoxStep) - and they should be optinal. If the user doesn't provide a list of class labels, SSDToBoundingBoxStepRunner should still work, but just returning null for the bounding box labels
(b) the field should be static final not just final (so public static final String[] COCO_CLASSES_LABELS on SSDToBoundingBoxStep is better)


This means:
1. We introduce a `protected String[] classLabels` in to SSDToBoundingBoxStep (default value: null)
2. We keep the coco labels ONLY as public static final field in SSDToBoundingBoxStep as a convenience to users - i.e., they can do `new SSDToBoundingBoxStep().classLabels(SSDToBoundingBoxStep.COCO_LABELS)`. It's not used as a default or anything.

Also if the labels aren't provided or are empty: we should simply have a null label for that bounding box.